src/billing/List/BillingList.tsx

```diff
@@ -155,158 +153,132 @@ export const BillingList = (props: BillingListProps) => {
     };
   };
 
-  return div({ role: 'main', style: { display: 'flex', flex: 1, height: `calc(100% - ${Style.topBarHeight}px)` } }, [
-    div(
-      {
-        style: {
+  const renderContent = (): ReactNode => {
```

Comment:
Moved this out into a render function because otherwise the conversion (removing Utils.cond) involved nested ternary blocks, which are not recommended.

Reply:
this would be better as a functional component with the needed props to live outside of this larger component.  That gives us better render performance, readability, and even code-reuse potential and unit testability.

Reply:
Done in https://github.com/DataBiosphere/terra-ui/pull/5028/commits/f2575ac09acca6ce407dc86e51021278ca3198d1.